### PR TITLE
U1-80 Write unit test for action show-settings

### PR DESCRIPTION
**Pull request by huyhqit**
JIRA ticket: https://pass-career.atlassian.net/projects/U1/issues/U1-80

### CODE REVIEW

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ source/actions/show-settings.js  80
         

Clones added
============
- tests/unit/actions/show-settings.test.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980411&bid=16448942)

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ source/actions/show-settings.js  85
         

Clones added
============
- tests/unit/actions/show-settings.test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980411&bid=16448942)

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Coverage increased per file
===========================
+ source/actions/show-settings.js  100
+ source/actions/get-images.js  4
         

Clones added
============
- tests/unit/actions/show-settings.test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980411&bid=16448942)

File: tests/unit/actions/show-settings.test.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      infoRecord += 'Exit';
+    });
+    // Get images
+    await getImages({}, answers, myLogger);
+    // Run show-settings
+    await showSettings({}, {}, myLogger);
+    // Expect output from function wait-key-to-exit
+    expect(infoRecord.includes(chalk.cyan('\nPress any key to exit..'))).toBeTruthy();
+    // Restore mock
+    console.log = oldConsoleLog;
```
Comment (knightus) [inline tests/unit/actions/show-settings.test.js]: Codacy Issue found: [Unexpected console statement.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980411)

File: tests/unit/actions/show-settings.test.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      path: folder,
+      orientation: ORIENTATION_LANDSCAPE,
+      namePattern: IMAGE_NAME_FORMAT_HASH,
+    };
+    // Mock process.argv and process title to trigger Enter key to exit
+    const oldProcessTitle = process.title;
+    process.title = `${process.cwd()}\\get-lock-screen.exe`;
+    const oldProcessArgv = process.argv;
+    process.argv = [...oldProcessArgv, '', ''];
+    const oldConsoleLog = console.log;
```
Comment (knightus) [inline tests/unit/actions/show-settings.test.js]: Codacy Issue found: [Unexpected console statement.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980411)

File: tests/unit/actions/show-settings.test.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    console.log = jest.fn((data) => {
```
Comment (knightus) [inline tests/unit/actions/show-settings.test.js]: Codacy Issue found: [Unexpected console statement.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980411)

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ source/actions/show-settings.js  100
+ source/actions/get-images.js  4
         

Clones added
============
- tests/unit/actions/show-settings.test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980411&bid=16448942)

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- tests/unit/actions/show-settings.test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4980411&bid=16448942)

**Review by duyphaphach — APPROVED** [review]

**Pull request merged**